spec/syntax.md

```diff
@@ -91,150 +91,198 @@ The syntax specification takes into account the following design restrictions:
 
 _This section is non-normative._
 
-### Messages
+### Messages and Patterns
 
-All messages, including simple ones, are enclosed in `{…}` delimiters:
+A **_message_** is the complete template for a specific message formatting request.
 
-    {Hello, world!}
+All _messages_ MUST contain either a _pattern_ or _selectors_. 
+A _message_ MAY contain zero or more _declarations_.
+(Implication: the empty string is not a valid message)
 
-The same message defined in a `.properties` file:
+A **_pattern_** is a combination of _text_ and _placeholders_ 
+to be formatted as a unit.
+All _patterns_, including simple ones, begin with U+007B LEFT CURLY BRACKET `{` 
+and end with U+007D RIGHT CURLY BRACKET `}`.
 
-```properties
-app.greetings.hello = {Hello, world!}
-```
+> A _message_ consisting of a simple _pattern_:
+>>```
+>>{Hello, world!}
+>>```
```

Comment:
There should be no need to double-indent conde samples. This applies to all other instances as well:

```suggestion
> A _message_ consisting of a simple _pattern_:
> ```
> {Hello, world!}
> ```
```

Reply:
Will do.

Reply:
Done in 5aef879